README.md

```diff
@@ -101,6 +101,19 @@ preflight version 0.0.0 <commit: 2d3bb671bff8a95d385621382f31215234877d44>
 
 [releases_link]:https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases
 
+## Preflight Testing
+
+For e2e testing, run
+
+```bash
+go test -v `go list ./... | grep -v e2e`
```

Comment:
There is discrepancy between the instruction and the provided command. The `-v` option of `grep` performs an inversion selection; So this line filters out the e2e tests, not running them.

Reply:
Ah I see and I made the changes. Thank you!